checker/src/main/java/org/checkerframework/checker/initialization/InitializationChecker.java

```diff
@@ -4,37 +4,112 @@
 import com.sun.source.tree.Tree;
 import com.sun.source.tree.VariableTree;
 
+import org.checkerframework.checker.initialization.InitializationFieldAccessAnnotatedTypeFactory.CommitmentFieldAccessTreeAnnotator;
+import org.checkerframework.checker.initialization.qual.HoldsForDefaultValues;
+import org.checkerframework.checker.initialization.qual.Initialized;
+import org.checkerframework.checker.nullness.NullnessChecker;
+import org.checkerframework.checker.nullness.NullnessNoInitAnnotatedTypeFactory;
+import org.checkerframework.checker.nullness.NullnessNoInitSubchecker;
+import org.checkerframework.checker.nullness.qual.EnsuresNonNull;
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
 import org.checkerframework.common.basetype.BaseTypeChecker;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.NavigableSet;
+import java.util.Set;
 
 /**
  * Tracks whether a value is initialized (all its fields are set), and checks that values are
  * initialized before being used. Implements the freedom-before-commitment scheme for
  * initialization, augmented by type frames.
  *
+ * <p>Because there is a cyclic dependency between this type system and the target type system,
+ * using this checker is more complex than for others. Specifically, the target checker must:
+ *
+ * <ol>
+ *   <li>Use a subclass of this checker as its parent checker. This is necessary because this
```

Comment:
I'm confused by how many checkers are involved in this sentence and how it relates to the Nullness Checker example below. Can you try to clarify? A checker can declared their subcheckers, but not their parent checkers, right? So maybe if you switched the descriptions to be in the order in which they are actually declared it would be clearer?